packages/nextra/src/server/compile.ts

```diff
@@ -153,6 +154,22 @@ export async function compileMdx(
   const format =
     _format === 'detect' ? (filePath.endsWith('.mdx') ? 'mdx' : 'md') : _format
 
+  const mathPlugin: PluggableList = []
+  if (latex) {
+    if (typeof latex === 'object') {
+      switch (latex.renderer) {
+        case 'katex':
+          mathPlugin.push(rehypeKatex)
+          break
+        case 'mathjax':
+          mathPlugin.push([rehypeReactMathjax, latex.options])
```

Comment:
Can you mention any example of which options we can pass? 🙏 so i could test it too

Reply:
Currently the config is not passed in, but here is an example of one of the options: https://github.com/fast-reflexes/better-react-mathjax#example-2-basic-example-with-asciimath  They switch out the usual parser for the asciimath parser